### PR TITLE
Added contributing.md

### LINKED ISSUE #3: Establish Contribution Guidelines
Either place in the wiki or within the .github/ directory.
Part of this will be requiring the use of the templates

### PR DESCRIPTION
**Pull request by itsercleung**
## Link to Issue Number
closes #3 

## Description
Added contributing.md which shows contribution guidelines including PR, code conventions, and other relevant information for first-time contributors to read

## Checklist
- [x] All feature request A/C have been met OR the bug has been fixed
- [x] I ran all nessisary tests and they pass
- [x] I rebased upstream/master onto my working branch before opening this PR
- [x] I have named my PR something sensible
- [x] I have included the issue number above
- [x] I have assigned atleast two people to review my changes

## Other Notes
No notes

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+- Becoming a maintainer
+
+## Pull Requests
+
+Please ensure these poitns are followed during PR:
```
Comment (nidhineshnand) [inline .github/CONTRIBUTING.md]: The spelling of points is wrong

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+Refer to this [link](https://github.com/SE701Group3/restaurant-manager/wiki/Git-Workflow) for in-detail pull request instructions.
+
+## Any contributions you make will be under the GNU v3.0 Software License
+
+In short, when you submit code changes, your submissions are understood to be under the same [GNU v3.0 License](https://www.gnu.org/licenses/gpl-3.0.en.html) that covers the project. Feel free to contact the maintainers if that's a concern.
+
+- By contributing, you agree that your contributions will be licensed under its GNU v3.0 License.
+
+## Report bugs using Github's [issues](https://github.com/SE701Group3/restaurant-manager/issues)
```
Comment (nidhineshnand) [inline .github/CONTRIBUTING.md]: Add the wiki link related to bug reporting

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+- Becoming a maintainer
+
+## Pull Requests
+
+Please ensure these poitns are followed during PR:
+
+- Ensure a clean commit history - make one commit per feature and squash relevant fixes if needed
```
Comment (gracecocks) [inline .github/CONTRIBUTING.md]: We should always squash the fixes to a single commit, so should read "and squash relevant fixes"

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+- Ensure a descriptive pull request - make sure to include relevant links, and relationships between the issue/feature you are doing
+- Ensure two or more PR review assignees
+- Do not accept your own pull request - after the PR review either the PR reviewees or admin will squash and merge your PR
```
Comment (gracecocks) [inline .github/CONTRIBUTING.md]: should this be 'do not **merge** your own pull request"?

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (nidhineshnand) [inline .github/CONTRIBUTING.md]: Update the link for the template

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (nidhineshnand) [inline .github/CONTRIBUTING.md]: The spelling of point is wrong

File: .github/CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+- Steps to reproduce
+  - Be specific!
+  - Give sample code if you can. [Example SO sample](http://stackoverflow.com/q/12488905/180626)
+- What you expected would happen
+- What actually happens
+- Notes (possibly including why you think this might be happening, or stuff you tried that didn't work)
+
+## Use a Consistent Coding Style
+
+Use the following wiki page for both React and C#: [Coding Conventions](https://github.com/SE701Group3/restaurant-manager/wiki/Coding-Conventions)
```
Comment (nidhineshnand) [inline .github/CONTRIBUTING.md]: Update link

**Pull request merged**